docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -57,6 +59,19 @@ Please note that no additional task/goal is required for the Gradle plugin.
                 </execution>
             </executions>
         </plugin>
+        <plugin>
+            <groupId>org.jboss.jandex</groupId>
+            <artifactId>jandex-maven-plugin</artifactId>
+            <version>1.1.0</version>
```

Comment:
I think we need to make this an attribute, otherwise it will get outdated quite fast.

This needs to be done in several places:
- in the build-parent/pom.xml, we need a property for the Jandex Maven plugin (you'll see it's already defined in this file with an hardcoded property)
- then in docs/pom.xml, you need to pass it as an attribute (you already have a couple of examples)

Do you want to do it or should I?

Thanks!

Reply:
Good point, I'll update those files in a bit

Reply:
done